src/app/(post-login)/submission/program-table/CancerTypes.tsx

```diff
@@ -18,14 +18,17 @@
  */
 'use client';
 
-const CancerTypes = ({ types }: { types: string[] }) => (
+import { Maybe } from '@/__generated__/graphql';
+
+const CancerTypes = ({ types }: { types: Maybe<Maybe<string>[]> | undefined }) => (
 	<div>
-		{types.map((cancerType, i) => (
-			<div key={cancerType}>
-				{cancerType}
-				{i < types.length - 1 && ','}
-			</div>
-		))}
+		{types &&
+			types.map((cancerType, i) => (
```

Comment:
Optional chaining is a good approach here too
```suggestion
		{types?.map?.((cancerType, i) => (
```

Reply:
Shrink 'em down, updated